src/mumble/LookConfig.ui

```diff
@@ -447,17 +447,36 @@
         </property>
        </widget>
       </item>
-      <item row="2" column="0" colspan="2">
-       <widget class="QCheckBox" name="qcbAskOnQuit">
+      <item row="2" column="0">
+       <widget class="QLabel" name="qliConnectedQuitBehavior">
+        <property name="text">
+         <string>Quit Behaviour</string>
```

Comment:
```suggestion
         <string>Quit Behavior</string>
```
The title seems to be a bit off though, as this is not about the general quit behavior, but only about the quit behavior while still connected to a server :eyes: 
I have the feeling that `Connected quit behavior` doesn't really cut it for the UI, though :thinking: